src/pages/partners/affiliate-ib/_calculated-formula.js

```diff
@@ -0,0 +1,186 @@
+                                    <FormulaValueResultSwapSynthetic>
+                                        <FormulaResultGreen>=</FormulaResultGreen>
+                                        {result.total}
+                                        {result.description && result.description}
+                                    </FormulaValueResultSwapSynthetic>
+                                </FormulaResult>
+                            </FormulaHighlight>
+                        </FormulaContainer>
+                    )
+                })}
+            </Show.Desktop>
+            <Show.Mobile>
+                <>
+                    {itemList.map((value, index) => {
+                        const totalItem = value.totalItem
+                        const formula = value.formula
+                        const totalResult = value.result.total
+                        const totalResultDesc = value.result.description
+                        const totalNumberofOperation = 2
+                        return totalItem > totalNumberofOperation ? (
+                            <FormulaContainer key={index}>
+                                <FormulaHighlight>
+                                    <FormulaTopWrapper>
+                                        {formula.map((list, indexData) => {
+                                            let items = list.item
+                                            let desc = list.description
+                                            let nextOperator = list.next_operator
+
+                                            return (
+                                                <>
+                                                    <FormulaValueMobile key={indexData}>
+                                                        {items}
+                                                        {indexData > 0 ? (
+                                                            <PointerContainer top>
+                                                                <PointerDot />
+                                                                <PointerStick />
+                                                                <PointerText top>
+                                                                    {desc}
+                                                                </PointerText>
+                                                            </PointerContainer>
+                                                        ) : (
+                                                            <PointerContainer>
+                                                                <PointerDot />
+                                                                <PointerStick />
+                                                                <PointerText>{desc}</PointerText>
+                                                            </PointerContainer>
+                                                        )}
```

Comment:
```suggestion
                                                        <PointerContainer top={indexData > 0}>
                                                                <PointerDot />
                                                                <PointerStick />
                                                                <PointerText top={indexData > 0}>
                                                                    {desc}
                                                                </PointerText>
                                                           </PointerContainer>
```

I think this can be simplified to the above. But gotta reconfirm 😄

Reply:
Amended in the latest commit